backend/routes/userRoutes.js

```diff
@@ -1,8 +1,9 @@
 const express = require('express');
 const router = express.Router();
+
 const {
     registerUser,
-    loginUser,
+    loginUser
```

Comment:
I actually don't know when this change was made. This could be left as the original.